src/topics/using-containers.md

```diff
@@ -52,8 +52,10 @@ used a container called `node:slim`.
 Provide a "hello.js" and invoke `cwltool` providing the tool description and the
 input object on the command line:
 
-```{code-block} console
-$ echo "console.log(\"Hello World\");" > hello.js
+```{literalinclude} /_include/cwl/using-containers/hello.js
```

Comment:
Thank you for your PR. This isn't rendering correctly at the moment because there is no `hello.js` file in https://github.com/khaiyra/user_guide/tree/hello.js-markdown/src/_includes/cwl

Perhaps you forgot to add the file to your branch?